### PR TITLE
fix(v1): registration "not unique" error message does not mention correct form label

### PR DESCRIPTION
**Pull request by sherizada-okta**
## Description:
Use correct form label value in error message.

Before:
Form label value is "Social Security Number", not "email".
reproErrorMessage

After:
Form label value is "Identifier". See placeholder in UI.
Screenshot 2021-06-16 at 2 07 02 PM
Screenshot 2021-06-16 at 2 06 47 PM

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-382511](https://oktainc.atlassian.net/browse/OKTA-382511)

### CODE REVIEW

File: src/RegistrationController.js [hunk 1]
```diff
@@ -112,16 +112,49 @@ export default BaseLoginController.extend({
...
+      () =>
+        this.model.trigger(
+          'error',
+          this.model,
+          {
+            responseJSON: { errorSummary: error.errorSummary }, // remove errorCauses so message renders at form-level
+          },
+          true
+        ),
+      100 // to beat race-condition with something that is clearing this error message
```
Comment (sherizada-okta) [inline src/RegistrationController.js:154]: I couldn't track down what was clearing. I listened to 'form:clear-errors' on the model and nothing seems to be calling it.

Comment (sherizada-okta) [reply src/RegistrationController.js:157]: I'd like to find a way to remove this `setTimeout`

Comment (sherizada-okta) [reply src/RegistrationController.js:157]: There's a race condition somewhere. If I have breakpoints set, I don't need this setTimeout.

Comment (sherizada-okta) [reply src/RegistrationController.js:157]: Race condition demo: https://okta.box.com/s/qbn4rb82ojiyb84a0b1mvi9rqh7nx33p

**Review by pradeepdewda-okta — APPROVED** [review]
I am good with this compromise..

**Pull request closed without merging**